### PR TITLE
Portfolio website Contact form

### PR DESCRIPTION
**Pull request by DhuruvSingh**
### Portfolio: add a contact form and finish the mobile version

- Added Input with Name field accepts up to 30 chars.
- Added Input with Email field accepts only emails.
- Added Input with Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).

> Please Review And Give Your Valuable Feedback

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -296,6 +296,45 @@ <h4>Skills</h4>
             </div>
         </section>
     </main>
+    <section id="contact-form">
+        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        <form action="https://formspree.io/f/xeqnpraa" method="post">
+            <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+            <input type="email" name="email" placeholder="Email address" required>
+            <textarea name="message" maxlength="500" placeholder="Enter text here"></textarea>
+            <button type="submit">Get in touch</button>
```
Comment (tufoinnkuo10) [inline index.html:305]: - [ ] Great job with the contact section. everything works as expected. nevertheless, kindly add the bottom margin so as to make it look more like the figma version
your version
Screenshot (207)
figma version 
Screenshot (208)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @DhuruvSingh 
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅form submits successfully
✅figma design well followed

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by muneebulrehman — APPROVED** [review]
Hi @DhuruvSingh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good use of semantic tags.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DhuruvSingh** [comment]
Thank you so much for your valuable feedback @muneebulrehman @tufoinnkuo10

**Pull request merged**